config/routes.rb

```diff
@@ -146,6 +146,13 @@
 
       scope module: :forum do
         resources :forums do
+          resources :topics do
+            post 'subscribe', on: :member
+            delete 'unsubscribe', on: :member
+            put 'locked', on: :member
+            put 'hidden', on: :member
```

Comment:
Yeah I know. I'm not saying that this is not correct. But I think use verb will have better code readability. 

When you first look at the controller and these actions, how do you know that these actions are putting a value from their names?